HIP/hip-0000-add-ledger-id-field.md

```diff
@@ -0,0 +1,575 @@
+- hip: <HIP number (this is determined by the HIP editor)>
+- title: Add Network ID Field to Info Responses
+- author: Simi Hunjan <simi@hedera.com>, David Matusevich <davidmatusevich@swirlds.com>
+- type: Standards
+- category: Services
```

Comment:
Following the rest of the naming, let's keep it singular rather than plural - `Service`